src/backend/api/Fusion.Resources.Api/Controllers/Requests/ApiModels/ApiRequestsTimeline.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Fusion.Resources.Domain;
+
+namespace Fusion.Resources.Api.Controllers
+{
+    public class ApiRequestsTimeline
+    {
+   
+        public ApiRequestsTimeline(QueryRequestsTimeline qt)
+        {
+            if (qt.Requests != null) Requests = qt.Requests.Select(r => new SimpleRequest(r)).ToList();
+            if (qt.Timeline != null) Timeline = qt.Timeline.Select(t => new RequestsTimelineRange(t)).ToList();
+        }
+        public List<SimpleRequest>? Requests { get; set; }
+        public List<RequestsTimelineRange>? Timeline { get; set; }
+        public class SimpleRequest
```

Comment:
Is the `SimpleRequest` class relevant outside `ApiRequestsTimeline`?

Reply:
I think it is relevant to just keep that one internal. 
Perhaps it is relevant for sector, but i guess it can be refactored at that point?